lib/twilio-ruby/http/http_client.rb

```diff
@@ -44,8 +44,8 @@ def _request(request)
         @last_response = nil
 
         response = send(request)
-        if response.status == 504
-          object = { message: 'Request timeout', code: 504 }.to_json
+        if Faraday::Response::RaiseError::ServerErrorStatuses.include?(response.status)
```

Comment:
`ServerErrorStatuses` is just a range `(500..600)`, perhaps you could use it here directly instead of referencing the class with its namespace from Faraday's  `RaiseError` ?

```ruby
if (500.600).include?(response.status)
```

Reply:
@januszm you're definitely right, I just replaced the Faraday dependency with `500..599` 👍